scripts/db_backup.sh

```diff
@@ -209,19 +222,19 @@ copy_db_to_azure_storage()
 {
     install-azure-cli
```

Comment:
this requires a lot of testing. in the interest of time and complexity: i vote we fix this with a separate pull request. i'll open a task in TFS